src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -106,7 +106,7 @@ pal::string_t fx_muxer_t::resolve_cli_version(const pal::string_t& global_json)
     catch (const web::json::json_exception& je)
     {
         pal::string_t jes = pal::to_palstring(je.what());
-        trace::error(_X("A JSON parsing exception occurred: %s"), jes.c_str());
+        trace::info(_X("A JSON parsing exception occurred: %s"), jes.c_str());
     }
     trace::verbose(_X("CLI version is [%s] in global json file [%s]"), retval.c_str(), global_json.c_str());
     return retval;
```

Comment:
If you want to turn that into an info, that is fine as a workaround. Please add a file name while you are at it, so you can close out your other issue.

`trace::info(_X("A JSON parsing exception occurred: %s"), jes.c_str());`
`-->`
`trace::info(_X("A JSON parsing exception occurred in %s: %s"), config_file.c_str(), jes.c_str());`